components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js

```diff
@@ -46,6 +46,19 @@ $(document).ready(function() {
 
 });
 
+function getLogStatusIcon(entry) {
+    switch (entry) {
+        case 'COMPLETED':
+            return 'fw-success';
+        case 'PENDING':
+            return 'fw-pending';
+        case 'ERROR':
+            return 'fw-error';
+        default:
+            return 'fw-info'
```

Comment:
Add semicolon by terminating the statement

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/pull/950